assets/benchmarks/shaders/CMakeLists.txt

```diff
@@ -84,4 +84,14 @@ generate_rules_for_shader("shader_benchmark_ps_alu_bound"
 generate_rules_for_shader("shader_benchmark_ps_mem_bound"
     SOURCE "${PPX_DIR}/assets/benchmarks/shaders/Benchmark_PsMemBound.hlsl"
     INCLUDES ${INCLUDE_FILES}
-    STAGES "ps")
\ No newline at end of file
+    STAGES "ps")
+
+generate_rules_for_shader("shader_benchmark_skybox"
```

Comment:
I assume this is until the other PR is merged.

Reply:
Yes, but I think we can keep this `shader_benchmark_skybox` since we are only planning to use the skybox to fill the background.

Reply:
oh yes, I meant you added this in the skybox PR, so this should not be shown as diff once rebased (there are conflicts right now)